lib/SILGen/SILGenType.cpp

```diff
@@ -1135,7 +1135,6 @@ class SILGenType : public TypeMemberVisitor<SILGenType> {
     // are existential and do not have witness tables.
     for (auto *conformance : theType->getLocalConformances(
                                ConformanceLookupKind::NonInherited)) {
-      assert(conformance->isComplete());
```

Comment:
Are we brave enough to remove isComplete() entirely yet? It serves no purpose as far as I can tell.

Reply:
There are about 9 remaining uses; some of them are used in logic I'm unfamiliar with so I'm not quite brave enough to remove it outright just yet but I think we're getting close.